src/schema/ghga.yaml

```diff
@@ -191,7 +192,7 @@ classes:
       type:
         description: >-
           Type of sample: 'Case' or 'Control'.
-        # TODO: Enumerate values for this slot via SampleType
+        range: case_control_enum
```

Comment:
Great! Could we have a better name instead of `case_control_enum`? If not, we can revisit this later on too.